src/models/searchservice/response/directanswer/fieldvaluedirectanswers/AddressDirectAnswer.ts

```diff
@@ -0,0 +1,29 @@
+import { BuiltInFieldType } from '../BuiltInFieldType';
+import { BaseFieldValueDirectAnswer } from '../BaseFieldValueDirectAnswer';
+import { EnumOrLiteral } from '../../../../utils/EnumOrLiteral';
+
+/**
+ * A {@link BaseFieldValueDirectAnswer} interface with 'address' field type.
+ *
+ * @public
+ */
+export interface AddressDirectAnswer extends BaseFieldValueDirectAnswer<Address> {
+  fieldType: EnumOrLiteral<BuiltInFieldType.Address>
+}
+
+/**
+ * An interface with address fields to use in {@link BaseFieldValueDirectAnswer.value}.
+ *
+ * @public
+ */
+export interface Address {
```

Comment:
you need to export this in the fieldvaluedirectanswer/index.ts file was well since this is part of the interface above

Reply:
I think it was added in the file. Can you check again if the file changes are refreshed?

Reply:
I don't see `Address` in the index file. Also this warning indicates it's not exported.
<img width="674" alt="Screen Shot 2022-08-16 at 3 26 32 PM" src="https://user-images.githubusercontent.com/36055303/184965195-dd4b93eb-0620-45a5-bbd6-fff1e915dcfe.png">

Reply:
maybe you didn't push your new changes?

Reply:
Oh I see!